airflow/executors/local_executor.py

```diff
@@ -349,7 +349,7 @@ def start(self) -> None:
         self.workers_active = 0
         self.impl = (
             LocalExecutor.UnlimitedParallelism(self)
-            if self.parallelism == 0
+            if self.parallelism <= 0
```

Comment:
I think it's better to raise exception that it should be 0 or bigger.

Reply:
@potiuk 
i changed code. it'll raise exception when self. parallelism less than 0